sci-libs/hdf/hdf-4.2.13-r1.ebuild

```diff
@@ -61,7 +61,7 @@ src_configure() {
 
 src_install() {
 	default
-	use static-libs || prune_libtool_files --all
+	use static-libs || find "${ED}" -name '*.la' -delete || die
 	dodoc release_notes/{RELEASE,HISTORY,bugs_fixed,misc_docs}.txt
```

Comment:
Convert these into an if for readability?

Reply:
I can do that. I just took it that way, as several ebuilds start using this way, to make it consistent. But I don't mind to change it to an full if.

Reply:
Done.